tests/Unit/Evolution/Systems/Cce/Test_ReadBoundaryDataH5.cpp

```diff
@@ -345,7 +345,8 @@ void test_data_manager_with_dummy_buffer_updater(
           time_buffer, solution, extraction_radius, amplitude, frequency, l_max}
           .get_clone(),
       l_max, buffer_size,
-      intrp::BarycentricRationalSpanInterpolator{3u, 4u}.get_clone()};
+      std::make_unique<intrp::BarycentricRationalSpanInterpolator>(
```

Comment:
fix the other one (i.e. two lines up use make_unique instead of get_clone)
This line can be simplified to 
`std::make_unique<intrp::BarycentricRationalSpanInterpolator>(3u, 4u)`